app/views/unattended/provisioning_templates/PXEGrub/pxegrub_default_local_boot.erb

```diff
@@ -4,8 +4,16 @@ name: PXEGrub default local boot
 model: ProvisioningTemplate
 %>
 <%# Used to boot provisioned hosts, do not associate or change the name. %>
-
-default=0
+<%
+  # Grub1 only supports numeric default statement, this allows conversion to number. First define
+  # array of directories we will search for EFI booloaders:
+  paths = ["fedora", "redhat", "centos", "debian", "ubuntu", "sles", "opensuse", "Microsoft", "EFI"]
+  # Read default setting but since "local" is missing, use the first path available.
+  default_setting = global_setting("default_pxe_item_local", paths.first)
```

Comment:
Btw. would it make sense to support `local_chain_hd0` value here, since we're rendering this in the template? Asking mainly because we mention in the help for the setting